healthcheck/templateHealthcheck.go

```diff
@@ -36,6 +37,7 @@ const TemplateHealthcheckHTML = `
             <th>Query</th>
             <th>Test Ran?</th>
             <th>Expected</th>
+						<th>Operation</th>
```

Comment:
Here too, too much indent?

Reply:
I don't know why my IDE (Atom) is not showing me these tabs in the right way